dotnet/src/SemanticKernel/SemanticFunctions/PromptTemplateConfig.cs

```diff
@@ -65,6 +66,13 @@ public class CompletionConfig
         [JsonPropertyOrder(6)]
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
         public List<string> StopSequences { get; set; } = new();
+
+        /// <summary>
+        /// The number of seconds to wait before the request to the completion backend times out.
+        /// </summary>
+        [JsonPropertyName("http_timeout_in_seconds")]
+        [JsonPropertyOrder(7)]
+        public int HttpTimeoutInSeconds { get; set; } = OpenAIClientAbstract.DEFAULT_HTTP_TIMEOUT_IN_SECONDS;
```

Comment:
I think this should be part of the PromptTemplateConfig but not the CompletionConfig -- the property could/would be relevant for Edit/Insert/etc. as well.